runtime/include/jcltracepoints.h

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2025
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ *******************************************************************************/
+
+ /* These definitions required by j9.h are in the OpenJ9 jni.h, but needed when OpenJDK jni.h is used. */
+struct GCStatus;
+typedef struct GCStatus GCStatus;
+struct JavaVMQuery;
+typedef struct JavaVMQuery JavaVMQuery;
+struct JVMExtensionInterface_;
+typedef const struct JVMExtensionInterface_ *JVMExt;
+
+#define COPY_PROGRESS_INFO_MASK 0
+#if defined(AIX)
+#define AIXPPC
+#define RS6000
+#endif /* defined(AIX) */
+#if defined(MACOSX)
+#define OSX
+#endif /* defined(MACOSX) */
+#if defined(WIN32)
+#define OMR_OS_WINDOWS
+#endif /* defined(WIN32) */
+
+#include "j9.h"
+#include "tracehelp.c"
```

Comment:
This deserves a comment explaining why a header file includes a C source file.

Reply:
On second thought, if including `tracehelp.c` works, perhaps it should just appear in the `.c` file that includes `jcltracepoints.h`? It still feels wrong to include `.c` files in a `.h` file.

Reply:
If I move tracehelp.c out, then jcltracepoints.h is no longer specific to tracepoints, but more generic for loading j9.h outside of OpenJ9. jcltracepoints may no longer be an appropriate name, does `includej9h.h` work for you?

Reply:
That seems somewhat awkward to me, but I'm not sure I have a better suggestion.

More troubling is the collection of configuration definitions: I think we should be including `j9cfg.h` instead (a number of source files in the extensions repos already do that).

Reply:
Are you talking about the parts like the following? I don't believe including j9cfg.h fixes these. The OpenJ9 definitions are on the command line, not defined in j9cfg.h afaik. I need to translate OpenJDK platform definitions to OpenJ9 definitions. Even j9cfg.h won't define the right thing unless these are set.
```
#if defined(AIX)
#define AIXPPC
#define RS6000
#endif /* defined(AIX) */
```

Reply:
Also note that j9.h does include j9cfg.h